client/state/plans/test/selectors.js

```diff
@@ -35,8 +39,20 @@ import deepFreeze from 'deep-freeze';
 /**
  * Internal dependencies
  */
-import { getPlans, isRequestingPlans, getPlan, getPlanRawPrice, getPlanSlug } from '../selectors';
+import {
+	getPlans,
+	isRequestingPlans,
+	getPlan,
+	getPlanRawPrice,
+	getPlanSlug,
+	getIntervalTypeFromCurrentPlan,
+} from '../selectors';
 import { PLANS, getStateInstance } from './fixture';
+import { TERM_ANNUALLY, TERM_BIENNIALLY, TERM_MONTHLY } from 'lib/plans/constants';
+import * as plansLib from 'lib/plans';
+
+plansLib.getPlan = jest.fn();
```

Comment:
Can we do this as part of the test preparation - either in the test itself, or in `beforeEach` if it applies to all tests?